bokehjs/package.json

```diff
@@ -24,96 +24,97 @@
     "browserify": "^16.2.2",
     "chai": "^3.5.0",
     "chai-as-promised": "^5.1.0",
-    "del": "^1.1.1",
+    "del": "^3.0.0",
     "istanbul": "^0.4.5",
     "gulp": "^3.9.0",
     "gulp-insert": "^0.5.0",
     "gulp-less": "^4.0.0",
-    "gulp-rename": "^1.2.0",
-    "gulp-sourcemaps": "^1.5.2",
+    "gulp-rename": "^1.2.3",
+    "gulp-sourcemaps": "^2.6.4",
     "gulp-task-listing": "^1.0.0",
-    "uglify-es": "^3.1.9",
+    "uglify-es": "^3.3.9",
     "gulp-uglify": "^3.0.0",
     "gulp-uglifycss": "^1.0.4",
     "gulp-util": "^3.0.4",
-    "gulp-typescript": "^3.2.3",
-    "gulp-coffee": "^2.3.1",
-    "gulp-tslint": "^8.1.2",
-    "merge2": "^1.2.0",
+    "gulp-typescript": "^4.0.2",
+    "gulp-coffee": "^3.0.2",
+    "gulp-tslint": "^8.1.3",
+    "merge2": "^1.2.2",
     "logtrap": "^0.0.1",
     "mocha": "^2.2.5",
-    "run-sequence": "^1.0.0",
+    "run-sequence": "^2.2.1",
     "sinon": "^1.15.4",
-    "vinyl-source-stream": "^1.1.0",
-    "yargs": "^3.11.0",
+    "vinyl-source-stream": "^2.0.0",
+    "yargs": "^11.0.0",
```

Comment:
I recently tried to upgrade yargs, and it completely broke compiler's build. Here it just works.

Reply:
After all it's not a mystery. I missed that you added `spawn-sync` to dependencies in "update flatbush". Reordering of items in `dependencies` didn't help to catch this, especially given that `spawn-sync` isn't mentioned anywhere in this PR. Will later inspect why this is needed.